src/iron.rs

```diff
@@ -64,6 +64,10 @@ impl<H: Handler> Iron<H> {
     /// This consumes the Iron instance, but does the listening on
     /// another task, so is not blocking.
     ///
+    /// The threading model used does not return a thread but rather a guard.
```

Comment:
This is the model used by all threading in rust, so it would be better to just say "This returns a guard, which will automatically..."